bundles/binding/org.openhab.binding.zwave/database/dragontech/pa100.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+    <Model>PA-100</Model>
+    <Label lang="en">Plug-in On/Off Switch</Label>
+    <CommandClasses>
+        <Class><id>0x20</id></Class>
+        <Class><id>0x25</id></Class>
+        <Class><id>0x27</id></Class>
+        <Class><id>0x59</id></Class>
+        <Class><id>0x5a</id></Class>
+        <Class><id>0x5e</id></Class>
+        <Class><id>0x70</id></Class>
+        <Class><id>0x72</id></Class>
+        <Class><id>0x73</id></Class>
+        <Class><id>0x7a</id></Class>
+        <Class><id>0x85</id></Class>
+        <Class><id>0x86</id></Class>
+    </CommandClasses>
+
+    <Configuration>
+        <Parameter>
+            <Index>3</Index>
+            <Type>list</Type>
+            <Default>0</Default>
+            <Size>1</Size>
+            <Label lang="en">LED Indicator</Label>
+            <Help lang="en">Controls LED behavior when switch state is on/off</Help>
+            <Item>
+                <Value>0</Value>
+                <Label lang="en">off/on</Label>
+            </Item>
+            <Item>
+                <Value>1</Value>
+                <Label lang="en">on/off</Label>
+            </Item>
+            <Item>
+                <Value>2</Value>
+                <Label lang="en">off</Label>
+            </Item>
+        </Parameter>
+    </Configuration>
+
+    <Associations>
+        <Group>
+            <Index>1</Index>
```

Comment:
Is this group one that you would expect to be linked to openhab? If you want it to send reports to the binding, you need to add ```<SetToController>true</SetToController>``` in the group.